presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/RowExpressionFormatter.java

```diff
@@ -35,9 +38,13 @@
 public final class RowExpressionFormatter
 {
     private final ConnectorSession session;
+    private final FunctionManager functionManager;
+    private final StandardFunctionResolution standardFunctionResolution;
 
-    public RowExpressionFormatter(ConnectorSession session)
+    public RowExpressionFormatter(ConnectorSession session, FunctionManager functionManager)
     {
+        this.functionManager = functionManager;
```

Comment:
add `requireNonNull` like what we do for `session`. Move the assignment after the one for `session` to make sure the order is the same as declared on the stack.